hutch_python/env_version.py

```diff
@@ -14,6 +15,11 @@
 _dev_ignore_list = ['ami', 'pdsapp']
 
 
+def not_ignored(path, ignores=_dev_ignore_list):
+    """ Return True if _dev_ignore_list isn't in path or empty"""
+    return not (any((s in path for s in ignores)) or not path)
```

Comment:
Did I get this right?
`not (x or not y)` -> `not x and y` -> `y and not x`

```suggestion
    return bool(path and not any(s in path for s in ignores))
```

Reply:
logic is hard